packages/lib/src/wrappers.ts

```diff
@@ -227,6 +227,16 @@ export function autometrics<F extends FunctionSig>(
   const counterObjectiveAttributes: Attributes = {};
   const histogramObjectiveAttributes: Attributes = {};
 
+  // NOTE - Gravel Gateway will reject two metrics of the same name if one of them has a subset of the attributes of the other
+  //        This means to be able to support functions that have objectives, as well as functions that don't, we need to
+  //        default to setting the objective_* labels to the empty string.
+  histogramObjectiveAttributes.objective_name = "";
```

Comment:
just to check: this won't affect how objectives/alerts are registered right? Just so it doesn't pick up the empty label values.

Reply:
yeah i double-checked with ivan. empty string is the same as not setting the label:

https://prometheus.io/docs/concepts/data_model/#:~:text=A%20label%20with%20an%20empty%20label%20value%20is%20considered%20equivalent%20to%20a%20label%20that%20does%20not%20exist